src/main/scala/zio/s3/settings.scala

```diff
@@ -73,8 +73,13 @@ object S3Credentials {
       )
     )(p => effectBlocking(p.resolveCredentials()))
 
+  val fromWebIdentity: ZIO[Blocking, InvalidCredentials, S3Credentials] =
```

Comment:
can you please provide a comment saying need to include dependency sts thank you

Reply:
done